mbuild/conversion.py

```diff
@@ -863,6 +870,9 @@ def save(compound,
         output_sdf.write(pybel_molecule)
         output_sdf.close()
 
+    elif extension in ['.gro', '.top']:  # ParmEd supported saver.
+        structure.save(filename, overwrite=overwrite,
+                       combine=combine, **kwargs)
```

Comment:
I am also not sure what the best practice is, but would it make more sense to have `combine` be a kwarg?

I could see issues where another backend might use the `combine` flag as well, which could lead to unexpected behavior.

Reply:
Yeah, that's valid. I think I can try add a `try except` block to do this

Reply:
the only down point with be it will not be included in the docstring since it is optional